recipes/safeint/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "3.24":
```

Comment:
Call me crazy but this is older then the last release 🤔 

![image](https://user-images.githubusercontent.com/16867443/179867980-349db8bc-b11f-4eb5-8352-81a7af9a2eef.png)


I think upstream just has a very bizarer convention

Reply:
3.24.x > 3.0.x, isn't it?

Reply:
Yes, it should be. Well, sometimes we see projects breaking all things in a minor version, I would not be surprised by people changing the version convention in middle a patch too ...